src/formatting/reorder.rs

```diff
@@ -227,19 +229,34 @@ fn rewrite_reorderable_items(
             if context.config.merge_imports() {
                 normalized_items = merge_use_trees(normalized_items);
             }
-            normalized_items.sort();
```

Comment:
I'll confess this gave me pause at first glance, but see why we had to wrap the sorting behind the `reorder_imports` check here now since we can get here even if `reorder_imports` false with non-Preservation group import strategies.